### ISSUE TITLE
TLU module hangs

### ISSUE THREAD

**Issue by DavidLP** [SiLab-Bonn/pyBAR#50]
Quite often the TLU module issues one trigger and than hangs until a MIO power cycle.

**Comment by laborleben** [comment]
More details, which input?

**Comment by laborleben** [comment]
It is very unlikely that the complete board hangs when the TLU FSM is stuck. What are the status LEDs telling? Is the USB interface frozen?

**Comment by DavidLP** [comment]
### More details

Configuration:
- Trigger input RX2
- Trigger from scintillator
- Trigger inverted
- Or trigger from TDC (loop through) does not really matter

Cross checks:
- Trigger / TDC signal is there
- All other scans still work (noise occupancy etc.), that do not require the trigger signal
- Veto does not matter
- There are no trigger words in the data stream
- Nothing (but TLU FSM) is  stuck

Fix:
- Reload FPGA firmware

**Comment by laborleben** [comment]
pls test with latest development branch.

**Comment by laborleben** [comment]
pls test latest master/development branch with basil v2.4.3. In v2.4.3 there was a minor fix for a bug which could lead to the problem described above.

A possible workaround is to load the FPGA firmware before every run. Change the "avoid_download" parameter in dut_configuration_mio.yaml to False (https://github.com/SiLab-Bonn/pyBAR/blob/master/pybar/dut_configuration_mio.yaml#L5):

```
USB:
    bit_file       : "mio.bit"
    board_id       : 
    avoid_download : False
```

**Comment by laborleben** [comment]
I haven't seen that in a while. Any update?

**Comment by DavidLP** [comment]
I do not measure so I do not know. Fine to me to close.

**Comment by laborleben** [comment]
So most likely the issue was fixed in basil 2.4.3 as stated above.
